### PR TITLE
SQS-347 | integration, synthetic monitoring tests for active orders

### PR DESCRIPTION
**Pull request by deividaspetraitis**
Implements integration, synthetic monitoring tests for the for active orders query. Additionally adds test file for locust.

## Summary by CodeRabbit

- **New Features**
  - Introduced a new class for handling active orderbook orders, enhancing integration testing capabilities.
  - Added functionality to fetch active orders through a dedicated service method and endpoint.

- **Tests**
  - Expanded test coverage for active orderbook orders, including latency performance metrics and response validation.
  - Introduced new test cases for the `/passthrough/active-orders` endpoint to ensure comprehensive testing.

- **Chores**
  - Minor formatting adjustments for improved readability in the test suite.

### CODE REVIEW

File: tests/test_synthetic_geo.py [hunk 1]
```diff
@@ -41,6 +41,11 @@ def test_synth_pools_filters(self, environment_url):
     def test_synth_passthrough_portfolio_assets(self, environment_url):
         run_test_portfolio_assets(environment_url)
 
+
+    # /passthrough/active--orders endpoint
+    def test_synth_passthrough_portfolio_assets(self, environment_url):
+        run_test_portfolio_assets(environment_url)
```
[the remark this answers is not in the document]
Comment (p0mvn) [reply tests/test_synthetic_geo.py]: This seems like a valid recommendation

File: locust/passthrough_orderbook_active_orders.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from locust import HttpUser, task
+
+# burner address for the integration tests
+addr = "osmo1jgz4xmaw9yk9pjxd4h8c2zs0r0vmgyn88s8t6l"
+
+
+class PassthroughOrderbookActiveOrders(HttpUser):
+    @task
+    def passthroughOrderbookActiveOrders(self):
+        self.client.get(f"/passthrough/active-orders?userOsmoAddress={addr}")
```
[the remark this answers is not in the document]
Comment (p0mvn) [reply locust/passthrough_orderbook_active_orders.py]: This looks like a good call out

File: locust/passthrough_orderbook_active_orders.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from locust import HttpUser, task
```
Comment (p0mvn) [inline locust/passthrough_orderbook_active_orders.py:1]: Could you please remove the old `orderbook_active_orders.py` test? Seems unnecessary now

File: tests/test_synthetic_geo.py [hunk 4]
```diff
@@ -41,6 +41,11 @@ def test_synth_pools_filters(self, environment_url):
     def test_synth_passthrough_portfolio_assets(self, environment_url):
         run_test_portfolio_assets(environment_url)
 
+
+    # /passthrough/active--orders endpoint
```
Comment (p0mvn) [inline tests/test_synthetic_geo.py]:
```suggestion
    # /passthrough/active-orders endpoint
```

File: tests/test_passthrough.py [hunk 5]
```diff
@@ -1,34 +1,63 @@
...
+
+        start_time = time.time()
+        response = sqs_service.get_active_orderbook_orders(address)
+        elapsed_time_ms = (time.time() - start_time) * 1000
+
+        assert EXPECTED_LATENCY_UPPER_BOUND_MS > elapsed_time_ms, f"Error: latency {elapsed_time_ms} exceeded {expected_latency_upper_bound_ms} ms"
+
+        resp = OrderbookActiveOrdersResponse(**response)
+
+        resp.validate(address)
```
Comment (p0mvn) [inline tests/test_passthrough.py:55]: I like how clean this is

**Review by p0mvn — APPROVED** [review]
LGTM overall but experienced this when running

```
================================================================ short test summary info =================================================================
FAILED tests/test_passthrough.py::TestPassthrough::test_active_orderbook_orders[http://localhost:9092] - NameError: name 'expected_latency_upper_bound_ms' is not defined
```

Let's please fix pre-merge

File: locust/passthrough_orderbook_active_orders.py (hunk 3 above)
Comment (deividaspetraitis) [reply locust/passthrough_orderbook_active_orders.py:1]: Removed, good catch! :+1:

File: locust/passthrough_orderbook_active_orders.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (deividaspetraitis) [reply locust/passthrough_orderbook_active_orders.py]: Updated as per suggestion. We should probably to update rest method names too with next PRs.

File: tests/test_synthetic_geo.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (deividaspetraitis) [reply tests/test_synthetic_geo.py]: Good catch! Fixed. :+1:

**Comment by deividaspetraitis** [comment]
All suggestions were addressed, merging. :rocket:

**Pull request merged**